libs/rtemodel/include/RteTarget.h

```diff
@@ -853,6 +853,17 @@ class RteTarget : public RteItem
   */
   RteDeviceProperty* GetDeviceEnvironment() const { return m_deviceEnvironment; }
 
+  /**
+   * @brief get the generator input file path
+   * @return generator input file
```

Comment:
the comment says the method returns file, but the method name is  GetGeneratorInputFilePath. 
It must be clear if returned string is a file or a directory,  is it absolute or relative.